Source/MQTTnet/Server/MqttServerOptionsBuilder.cs

```diff
@@ -2,6 +2,7 @@
 using System.Net;
 using System.Net.Security;
 using System.Security.Authentication;
+using System.Security.Cryptography.X509Certificates;
```

Comment:
Does this also work for UWP? The code below is wrapped in an compiler statement but namespace is not.